src/components/AddItemForm/AddItemForm.js

```diff
@@ -82,6 +84,7 @@ const AddItemForm = ({ listId }) => {
       purchaseInterval: Number(formValues.purchaseInterval),
       lastPurchaseDate: null,
       numberOfPurchases: 0,
+      createdAt: firebase.firestore.FieldValue.serverTimestamp(),
```

Comment:
Do we want to set the createdAt date in the same way we are setting `lastPurchaseDate` in ShoppingList.js on line 49? I know there was some debate on keeping the times consistent.

```js
    db.collection(`lists/${listId}/items`)
      .doc(item.id)
      .update({
        lastPurchaseDate: new firebase.firestore.Timestamp(
          newPurchaseDate.toSeconds(),
          0,
        ),
        numberOfPurchases: firebase.firestore.FieldValue.increment(1),
        purchaseInterval: newPurchaseInterval,
      })
      .catch((err) => {
        console.log(err);
      });
  };
```

Reply:
I think since we only use the createdAt timestamp in one place (setting it in the database) and don't have a JavaScript Date or Luxon DateTime object already in use with this value, we're fine using the serverTimestamp.